src/syntax/expressions/expression.jl

```diff
@@ -1,28 +1,33 @@
 struct Expression{N,A<:AbstractOperator} <: AbstractExpression
-	x::NTuple{N,Variable}
-	L::A
-	function Expression{N}(x::NTuple{N,Variable}, L::A) where {N,A<:AbstractOperator}
-
-		# checks on L
-		ndoms(L,1) > 1 && throw(ArgumentError(
-        "cannot create expression with LinearOperator with `ndoms(L,1) > 1`"))
+  x::NTuple{N,Variable}
+  L::A
+  function Expression{N}(x::NTuple{N,Variable}, L::A) where {N,A<:AbstractOperator}
+    # checks on L
+    ndoms(L,1) > 1 && throw(ArgumentError(
+                                          "Cannot create expression with LinearOperator with `ndoms(L,1) > 1`"))
```

Comment:
In these cases, I would keep the indentation as it was, since that gives the real saving in terms of columns. But I would close the parenthesis on a new line, like in the following 

```
my_function_call(
    arg1, arg2, arg3
)
```

But consider this a non-issue, we can do a pass to clean up the code style later.

Reply:
yes I think we should, I think there is still some strange indentation in some of the code...